tests/generation/UtilitiesTest.py

```diff
@@ -0,0 +1,69 @@
+import inspect
```

Comment:
We can merge this commit with the very first commit of the PR.

Reply:
👍